system-tables/system-table-sql-diagnostics.md

```diff
@@ -2,7 +2,7 @@
 title: SQL 诊断
 summary: 了解 SQL 诊断功能。
 category: reference
-aliases: ['/docs-cn/dev/reference/system-databases/sql-diagnosis/','/docs-cn/dev/system-tables/system-table-sql-diagnosis/']
+aliases: ['/docs-cn/dev/reference/system-databases/sql-diagnosis/','/docs-cn/dev/system-tables/system-table-sql-diagnosis/','/zh/tidb/dev/check-cluster-status-using-sql-statements']
```

Comment:
Added the `'/docs-cn/dev/reference/performance/check-cluster-status-using-sql-statements/'` alias back in https://github.com/pingcap/docs-cn/pull/4076